[AI gateway] Variety of tweaks / reformatting and reorganizing (#14674)
cloudflare/cloudflare-docs: content/ai-gateway/get-started/connecting-applications.md

{{<resource-group>}}

> {{<resource header="Observe and control" href="/ai-gateway/get-started/configuring-settings/" icon="learning-center-book">}} Observe and control your AI applications {{</resource>}}

Reviewer: ```suggestion
Learn how to [observe and control your AI applications](/ai-gateway/get-started/configuring-settings/).
```

Reviewer: I'd remove this formating bc I've never seen a **Next steps** with it, outside of the Overview page.
